src/Zaaksysteem/Entities/Zaak.php

```diff
@@ -45,6 +45,7 @@ class Zaak extends Entity
 
         'statussen' => Casts\Related\Statussen::class,
         'zaakinformatieobjecten' => Casts\Related\Zaakinformatieobjecten::class,
+        'rollen'    => Casts\Related\Rollen::class,
```

Comment:
Mierenneuken maar ik vind het mooier als er 1 spatie gebruikt wordt en het is ook beter te onderhouden als er nieuwe items toegevoegd worden.

Reply:
Niks mierenneuken, terechte opmerking :)